### PR TITLE
Fixposium

### PR DESCRIPTION
**Pull request by guidezpl**
What started out as a single bug fix ended up being a slew of fixes, so bear with me. This will enable the 2.10.0 release to the Play Store and [gallery.flutter.dev,](https://gallery.flutter.dev,) which were last updated in May 2021.

## Description

This PR includes these changes:

- Fixes https://github.com/flutter/gallery/issues/755. Reduce reliance on `FadeInImagePlaceholder` and redo the mobile and desktop carousels' layout and peeking behavior to be more pleasant
- Fixes https://github.com/flutter/gallery/issues/576
- Improves animation for settings panel. Was janky. Now differentiates speed between mobile (slow) and desktop (fast), and adjust some curves for pleasantness
- Fixes divider color for settings panel
- Fixes focus border and color for Crane, it was displayed on the inside and too dark
- Fixes https://github.com/flutter/gallery/issues/381

This PR is staged at https://gallery-flutter-staging.web.app.

## Tests
 - Goldens test updated

### CODE REVIEW

**Comment by guidezpl** [comment]
Speaking with @pennzht about the benchmark check, otherwise this is ready for review.

File: lib/pages/backdrop.dart [hunk 1]
```diff
@@ -44,7 +46,9 @@ class _BackdropState extends State<Backdrop> with TickerProviderStateMixin {
     super.initState();
     _settingsPanelController = AnimationController(
       vsync: this,
-      duration: const Duration(milliseconds: 200),
+      duration: widget.isDesktop
+          ? const Duration(milliseconds: 200)
```
Comment (esouthren) [inline lib/pages/backdrop.dart:50]: Should these be added to the constants file?

File: lib/pages/home.dart [hunk 2]
```diff
@@ -793,15 +793,13 @@ class _CarouselState extends State<_Carousel>
         }
         // We want the peeking cards to be 160 in height and 0.38 helps
         // achieve that.
-        value = (1 - (value.abs() * .38)).clamp(0, 1).toDouble();
+        value = (1 - (value.abs() * .3)).clamp(0, 1).toDouble();
```
Comment (esouthren) [inline lib/pages/home.dart:796]: Update docs to 0.3?

**Comment by guidezpl** [comment]
> Should these be added to the constants file? 

Good idea

File: lib/pages/home.dart (hunk 2 above)
Comment (guidezpl) [reply lib/pages/home.dart:796]: Done.

File: lib/pages/backdrop.dart (hunk 1 above)
Comment (guidezpl) [reply lib/pages/backdrop.dart:50]: Good idea, done

**Comment by pennzht** [comment]
Debugging. Issue could be coming from `package:transparent_image`

**Review by esouthren — APPROVED** [review]
LGTM, thanks for updates

**Comment by guidezpl** [comment]
> Debugging. Issue could be coming from `package:transparent_image`

FWIW, that package just contains a single constant https://github.com/brianegan/transparent_image/blob/master/lib/transparent_image.dart#L5

**Comment by guidezpl** [comment]
The first issue will be resolved by https://github.com/brianegan/transparent_image/pull/13. The second is blocking, https://github.com/flutter/gallery/issues/903, and affects the gallery at `main` with the specified browser. So I'll disable the test partially, pending a resolution and @yjbanov being back.

**Pull request merged**